applications/SolidMechanicsApplication/solid_mechanics_application.cpp

```diff
@@ -869,8 +869,39 @@ void KratosSolidMechanicsApplication::Register() {
     KRATOS_REGISTER_CONDITION(
         "SurfaceElasticCondition3D9N", mSurfaceElasticCondition3D9N)
 
-    //Register Constitutive Laws
+    //Register Constitutive Laws with KRATOS_REGISTER to the Kernel
+    //Linear Elastic laws
+    KRATOS_REGISTER_CONSTITUTIVE_LAW("LinearElastic3DLaw", mLinearElastic3DLaw);
```

Comment:
@josep-m-carbonell as I have confirmed with you yesterday, I will add the following registers so that the read_material_process can detect the name of the constitutive law.